javascript/javascript3/week2/visual.js

```diff
@@ -0,0 +1,63 @@
+//target move one by one
+async function translateOneByOne() {
+    await new Promise((resolve) => {
+      resolve(
+        moveElement(document.querySelector("ul.marks li:nth-child(1)"), {
+          x: 20,
+          y: 300,
+        })
+      );
+      console.log("first");
+    });
+    await new Promise((resolve) => {
+      resolve(
+        moveElement(document.querySelector("ul.marks li:nth-child(2)"), {
+          x: 400,
+          y: 300,
+        })
+      );
+      console.log("first");
```

Comment:
Inside console is it second ??

Reply:
hey soujanya ...Thanks for reviewing my hw.. this console.log("first") have no meaning ...its for me to debug my code is working or not...i will remove it ...